### ISSUE TITLE
workflow for i18n

### ISSUE THREAD

**Issue by steida** [este/este-cli#40]
- allow export all dictionaries into one dir, so it can be sent to translators
- import translated dictionaries back to app, preserve new keys

**Comment by grabbou** [comment]
Yep. First I need to adjust translate generator to follow new rules, but yep.
